src/styles/colors.stories.tsx

```diff
@@ -1,35 +1,35 @@
-// @flow
 import * as React from 'react';
-import * as colors from './variables';
+import * as variables from './variables';
 import mdNotes from './colors.md';
 
-const bdlColors = {};
+const bdlColors: Record<string, Array<any>> = {}; // eslint-disable-line @typescript-eslint/no-explicit-any
```

Comment:
Should we consider disabling the `@typescript-eslint/no-explicit-any` rule for now? I've already run into some trouble with it in box-content-preview, as well.